cmd/influxd/launcher/cmd.go

```diff
@@ -451,6 +451,17 @@ func (o *InfluxdOpts) BindCliOpts() []cli.Opt {
 			Flag:  "storage-wal-fsync-delay",
 			Desc:  "The amount of time that a write will wait before fsyncing. A duration greater than 0 can be used to batch up multiple fsync calls. This is useful for slower disks or when WAL write contention is seen.",
 		},
+		{
+			DestP: &o.StorageConfig.Data.WALMaxConcurrentWrites,
+			Flag: "storage-wal-max-concurrent-writes",
+			Desc: "The max number of writes that will attempt to write to the WAL at a time. (default <nprocs> * 2)",
```

Comment:
Where does this default value get set in the code? I figured it would be somewhere with this commit but couldn't find it anywhere.

Reply:
The default-check happens here: https://github.com/influxdata/influxdb/pull/21621/files#diff-ffea3fc826d0307d8741d5e05848651828790cb749a839312fdc40ed2580677dR129-R131

The actual definition of the default is here: https://github.com/influxdata/influxdb/pull/21621/files#diff-ffea3fc826d0307d8741d5e05848651828790cb749a839312fdc40ed2580677dR69